### PR TITLE
Use : to split Assembly and Resource

### PR DESCRIPTION
**Pull request by Shawdooow**
- [ ] Update Samples?
- [ ] Test Samples?

### CODE REVIEW

**Comment by Shawdooow** [comment]
Actually with `:` now there is no need for `#`, gonna fix that first.

**Comment by jpbruyere** [comment]
I want to keep both mechanismes: in all case, embedded resource should start with `#` (I use that in other projects), and assembly searching is chosen depending on the presence of `:` or not. if not present, keep the existing mechanism.

File: vke/src/Utils.cs [hunk 1]
```diff
@@ -37,22 +37,25 @@ public static XmlAttributeOverrides GetXmlOverrides ()
...
+				string[] assemblyNames = resId.Split (':');
 				Assembly assembly = AppDomain.CurrentDomain.GetAssemblies ().FirstOrDefault (aa => aa.GetName ().Name == assemblyNames[0]);
-				if (assembly == null && assemblyNames.Length > 3)
-					assembly = AppDomain.CurrentDomain.GetAssemblies ()
-						.FirstOrDefault (aa => aa.GetName ().Name == $"{assemblyNames[0]}.{assemblyNames[1]}");
-				if (assembly != null && tryFindResource (assembly, resId, out stream))
-					return stream;
+				if (assembly == null)
+					throw new Exception("Assembly not found: " + path);
+                else
```
Comment (jpbruyere) [inline vke/src/Utils.cs]: no `else` needed after `throw`

**Comment by jpbruyere** [comment]
I've merged, but I reintroduce `#` prefix in any case, thank your for the id.

**Pull request merged**